homeassistant/components/aussie_broadband/sensor.py

```diff
@@ -12,8 +12,9 @@
 )
 
 from . import DOMAIN
+from ...const import DATA_MEGABYTES
 from ...core import HomeAssistant
-from ..sensor import SensorEntity
+from ..sensor import STATE_CLASS_TOTAL_INCREASING, SensorEntity
```

Comment:
```
# The state represents a monotonically increasing total, e.g. an amount of consumed gas
STATE_CLASS_TOTAL_INCREASING: Final = "total_increasing"
```

Since this does reset monthly, is it truly monotonic increasing?

Reply:
Yeah, so by definition, when a total_increasing entity goes down, its assumed to have reset. This means its possible for Home Assistant to show download usage by day/week/month irrespective of the billing cycling. This was only just introduced into dev recently.

Reply:
Excellent! This is awesome 💯